src/fmu/jobs/__init__.py

```diff
@@ -0,0 +1,2 @@
+# This line ONLY! :
```

Comment:
This will make the installed package `fmu.jobs `which seems much more general than probably the intention is. Better with `fmu.dataio.jobs`? Now it is like this:
```
:~/venv/3.8_fmu-dataio/lib/python3.8/site-packages/fmu$ ll
total 44
drwxrwxr-x   6 jan jan  4096 Jan 11 09:12 ./
drwxrwxr-x 424 jan jan 20480 Jan 11 09:12 ../
drwxrwxr-x   4 jan jan  4096 Jan 11 09:12 dataio/
drwxrwxr-x   4 jan jan  4096 Oct 12 19:23 ensemble/
-rw-rw-r--   1 jan jan    85 Jan 11 09:12 __init__.py
drwxrwxr-x   4 jan jan  4096 Jan 11 09:12 jobs/
drwxrwxr-x   2 jan jan  4096 Jan 11 09:12 __pycache__/
```

Reply:
Yes, that could be. Not sure. I see that in fmu-sumo it has been placed further up the structure (next to `fmu`). https://github.com/equinor/fmu-sumo/tree/master/src

Perhaps @berland knows the best practice of where to place the `jobs` folder?

Reply:
Subscript: No `jobs` parent folder, just `hook_implementations` directly under `subscript`. Perhaps the `jobs` folder is not needed here either. https://github.com/equinor/subscript/tree/master/src/subscript/hook_implementations

Reply:
Ecl2df also has this pattern, with `hook_implementations` directly under `ecl2df`: https://github.com/equinor/ecl2df/tree/master/ecl2df

Reply:
I copied the setup from subscript and ecl2df, more or less: Removed the `jobs` folder level and moved `hook_implementations` into `dataio`.

Reply:
Probably good enough to copy ecl2df/subscript yes.